src/components/Button/CreateKeywordButton.jsx

```diff
@@ -1,14 +1,20 @@
 import Button from "../UI/Button";
+import PropTypes from "prop-types";
 
-const CreateKeywordButton = () => {
+const CreateKeywordButton = ({ disabled }) => {
```

Comment:
`isPending` 동안 버튼을 disabled하기 위해서 해당 prop을 만들었습니다. 하지만 지금은 로딩 gif 파일로 form태그를 대체한 상황이라서, 추후 필요가 없는 경우 지우도록 하겠습니다.

Reply:
```suggestion
const CreateKeywordButton = ({ isDisabled }) => {
// (...)
  return (
    <Button
      type="submit"
      styles="flex-center px-14 py-8 font-medium border-2 border-purple-200 bg-purple-400/80 rounded-[15px] text-white text-18 hover:bg-purple-500/80"
      isDisabled={true}
    >
      생성하기
    </Button>
  );
}
```
해당 prop의 타입을 Boolean으로 관리하고, 이름도 `isDisabled`으로 수정해보면 어떨까요?
버튼 활성화도 토글이 되어서 '활성화 <> 비활성화' 전환이 되어야 할 것 같은데, 이때 Boolean으로 제어하면 직관적일 것 같습니다.
논의 후 수정된다면 `Button` 등 해당 prop과 관련한 컴포넌트들에 반영이 필요하겠습니다.

Reply:
이미 Boolean 값으로 관리되고 있군요. 제가 착각했는데, 이름만 수정 검토해주시면 감사하겠습니다 :)

Reply:
네 변수명만 보고 `boolean`으로 유추 가능할 것 같네요! 수정할게요 :)

Reply:
음,, disabled는 저희가 속성으로 쓰는 네이밍이라 바로 boolean이라고 유추가 가능할 것 같은데요, 우선 통일성을 위해서 앞에 is를 붙이도록 하겠습니다

Reply:
그렇군요. 이 맥락 까지 고려해보면, 기존처럼 disable을 유지하되 `isDisabled`으로 해보면 어때요~? `disabled` 속성에 쓰인다는 코드를 확인했을 때에만 bool 값 유추가 되었어요!

Reply:
변경했을 때, 적용되는 코드들을 보니 isEnabled 보단 isDisabled가 더 직관적인 것 같아요.
`disabled` 라는 속성에서 `disabled={!isEnabled}` 해당 코드의 가독성을 판단했을 때 한 번 더 생각을 해봐야 하는 것 같아서
기존의 `disabled` 보단 `isDisabled` 로 변경하는 것이 더 좋을 것 같은데 어떻게 생각하실까요? @SsongQ-92 @jin-ttao

Reply:
`isDisabled`로 수정하겠습니다 :)

Reply:
`isDisabled`로 수정 완료하였습니다~ :)